src/utils/structuredData.ts

```diff
@@ -3,9 +3,9 @@ import { secondsToISO8601 } from './datetime';
 
 import type { Playlist, PlaylistItem } from '#types/playlist';
 
-export const generateSeriesMetadata = (seriesPlaylist: Playlist) => {
-  // @todo this still used the series route
-  const seriesCanonical = `${window.location.origin}/s/${seriesPlaylist.feedid}`;
+export const generateSeriesMetadata = (seriesPlaylist: Playlist, mediaId: string | undefined) => {
```

Comment:
Also, I know that mediaID and seriesID are equivalent, but I would still call the param for this `seriesId` and probably better to just pass 1 param and let the caller do the `seriesPlaylist.feedid || mediaId` logic.

Reply:
Used one `seriesId` here.